sdk/spring/azure-spring-boot/pom.xml

```diff
@@ -251,6 +251,12 @@
       <version>1.1.1</version> <!-- {x-version-update;pl.pragmatists:JUnitParams;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.0.4</version>
```

Comment:
Shouldn't this be using the latest GA version available which is `1.0.7`?

Reply:
Yes, this snippet didn't be modified by rebasing and I will delete it manually.